exonum-java-binding-core/pom.xml

```diff
@@ -251,6 +252,17 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>osx</id>
```

Comment:
Please give the profile a concise yet meaningful name that describes its purpose: (e.g. `set-cargo-library-path-on-mac`) + document the profile.